lib/windows.js

```diff
@@ -68,6 +73,44 @@ const kernel32 = new ffi.Library('kernel32', {
 	QueryFullProcessImageNameW: ['int', ['pointer', 'uint32', 'pointer', 'pointer']]
 });
 
+function getProcessPathAndName(processHandle) {
+
+	if (ref.isNull(processHandle)) {
+		return [undefined, undefined];
+	}
+
+	// Set the path length to more than the Windows extended-length MAX_PATH length
+	const pathLengthBytes = 66000;
```

Comment:
Can you get this from a constant instead of hard-coding it?

Reply:
This was ripped from the main code base, not sure.

Reply:
At least try to Google it.

It's also not clear to me why it needs to be so much more than "extended-length MAX_PATH length", which is ~32000.